app/src/main/java/org/mozilla/focus/browser/integration/BrowserMenuController.kt

```diff
@@ -60,6 +63,8 @@ class BrowserMenuController(
                 }
             }
             is ToolbarMenu.Item.RemoveFromShortcuts -> {
+                Shortcuts.shortcutRemovedCounter["removed_from_browser_menu"].add()
```

Comment:
I wish these labels were generated by Glean from the yaml so we don't have to worry about mistakes in it.

Reply:
+1

Reply:
Noted!  https://bugzilla.mozilla.org/show_bug.cgi?id=1727551

Reply:
Sorry, I should have filed a bug about it immediately. :)